test/tests/long.py

```diff
@@ -146,3 +146,14 @@ class long2(long):
 
 print(long(x=10))
 print(long(x="10", base=10))
+
+try:
+    print(long('hek2mgl', 22))
+except Exception as e:
+    print(e.message)
+
+print(long('hek2mgl', 23))
+print(long('hek2mgl', 24))
+
+for i in range(-10, 10):
+    print(hash((-1 << 63) + i))
```

Comment:
can we also add `print(hash((1 << 63) + i))`? I think we get those a little bit different